### ISSUE TITLE
A11y_.NET Core_WPF_ListofProducts_AddProductListing_Zoom: The "Add Product" Button is not visible at 200% zoom.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/wpf#6734]
![6734](https://user-images.githubusercontent.com/86710491/176371946-3c28208b-630a-4c5a-823f-8710eaa7d916.png)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#.NET Core[WPF]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.4;#DesktopApp;#Win11;#Zoom;#Benchmark;

### Environment Details:
.NET Core
WPF
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Launch VS 2019 Int Preview
2. Navigate to Sample Applications and right-click on the DatabindingDemo and click on build.
3. Then right click on DatabindingDemo then Click on Debug-->Start New Instance. 
4. List of products screen should open.
5.Navigate to Add product button and click on it.
6. Zoom the screen to 200% and observe that The "Add Product" Button is not visible at 200% zoom

### Actual:
The "Add Product" Button is not visible at 200% zoom.

### Expected:
The "Add Product" Button should be visible at 200% zoom.

### User Impact:
Low vision users won't be able to see the add product button.

**Comment by merriemcgaw** [comment]
@Truptik12 what screen size are you setting for this test?

**Comment by Truptik12** [comment]
@merriemcgaw 

We are using the resolution 1366*768 and zooming the screen to 200% using custom scaling from the system settings.

**Comment by merriemcgaw** [comment]
@pchaurasia14 - this is something that would likely be adjusted in the sample app rather than WPF itself in my opinion. You reduce the size of the product list enough that the buttons are always showing.

**Comment by anjalisheel-wpf** [comment]
@merriemcgaw We have custom scaling between the range 100-500, so it may work fine if we address it by removing some items from the list, but the issue will be persisting for other custom scaling values.

**Comment by merriemcgaw** [comment]
@anjalisheel-wpf The current MAS guidelines specify that the tester should use 1280px and then do 200% scaling from there. As long as that enables all buttons etc to be shown, then you should be good for this issue.

**Comment by merriemcgaw** [comment]
@dipeshmsft - why is this issue closed? Did the sample app get updated, and did @Truptik12 validate the changes? Or was this something Office Hours said could be closed?

**Comment by dipeshmsft** [comment]
@Truptik12 , we can go ahead and test it again, we have merged the required changes, in the application

**Comment by pchaurasia14** [comment]
@Truptik12  - Still awaiting response on this.

**Comment by Truptik12** [comment]
@pchaurasia14 

Verified the bug in the latest VS version
Microsoft Visual Studio Professional 2022
Version 17.4.0 Preview 1.0

The bug is still reproducing.
![Screenshot (1207)](https://user-images.githubusercontent.com/86710491/186854859-d4f5b808-6160-41ad-bf73-3857b9941ea7.png)

**Comment by pchaurasia14** [comment]
@Truptik12  - Have you taken the latest commit for the application before testing ?

**Comment by merriemcgaw** [comment]
@Truptik12 it looks like you are going to want to get the latest version of the sample app to build and test. This looks like you were testing the old sample app.

**Comment by Truptik12** [comment]
@merriemcgaw @pchaurasia14 

Could you please provide the latest commit for testing?

**Comment by dipeshmsft** [comment]
https://github.com/Microsoft/WPF-Samples , you can clone the repo from here , or alternatively , download the zip from https://github.com/microsoft/WPF-Samples/archive/refs/heads/main.zip

**Comment by merriemcgaw** [comment]
Thanks @dipeshmsft!

**Comment by pchaurasia14** [comment]
@Truptik12  - Could you please update us on the status of this issue

**Comment by Truptik12** [comment]
@pchaurasia14 @dipeshmsft 

Tried to verify the bug using the latest commit, but we are getting the below error which says "'C:\Users\v-tkulkarni\Downloads\WPF-Samples-main\WPF-Samples-main\Sample Applications\DataBindingDemo\obj\project.assets.json' not found. Run a NuGet package restore to generate this file.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped =========="

![image](https://user-images.githubusercontent.com/86710491/188847524-b54c11e7-36b9-4e7b-8dfd-2216135055c7.png)

![image](https://user-images.githubusercontent.com/86710491/188848208-270357c1-9c24-40e1-b79a-e5980936b7a9.png)

We are unable to build the solution.

**Comment by miloush** [comment]
> Run a NuGet package restore to generate this file.

did you try that?

**Comment by pchaurasia14** [comment]
@Truptik12 - Any updates on this?

**Comment by Truptik12** [comment]
Tried to verify the bug using the latest commit, after running a NuGet package restore, but we are still getting the same error which says "'C:\Users\v-tkulkarni\Downloads\WPF-Samples-main\WPF-Samples-main\Sample Applications\DataBindingDemo\obj\project.assets.json' not found.
Could you please try to add the json file to the folder.

**Comment by pchaurasia14** [comment]
@Truptik12 - That json file will be automatically generated during build. Here are some options you can try - 

1. Try manually deleting bin & obj folders for this project and do a clean/rebuild. 
2. Do a fresh clone of the solution and try building
3. _Extreme case_ - Check if this is working on a clean/fresh machine  

Let us know if none of them work.

**Comment by Truptik12** [comment]
Hi @pchaurasia14 

We tried all the above instances, but we are still unable to get the screen. Could you please help us here.

**Comment by singhashish-wpf** [comment]
@Truptik12 
The error which you are mentioning is seen to occur if there is a mismatch of the version of dotnet installed and the one mentioned in the global.json. Could you try matching both of those, either installing the written version or updating the global.json with the installed dotnet version?

Are you doing it from Visual studio? You can instead just go to that folder and "dotnet build" and then run the exe inside the bin directory. If you want the exe, we can directly provide you with that.

**Comment by merriemcgaw** [comment]
> If you want the exe, we can directly provide you with that.

@singhashish-wpf if you can go ahead and provide the EXE as soon as possible that will help out a bunch! I'm asking for help getting your bugs validated ASAP, so if you can get that posted it would be greatly appreciated.

**Comment by Truptik12** [comment]
@singhashish-wpf 

Could you please provide the exe.

**Comment by dipeshmsft** [comment]
[DataBindingDemo.zip.zip](https://github.com/dotnet/wpf/files/9651764/DataBindingDemo.zip.zip)
Attaching the DataBindingDemo binaries.

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached

![image](https://user-images.githubusercontent.com/86710491/192529952-3dce846f-a723-4433-ab51-73522df960c0.png)
